.travis.yml

```diff
@@ -5,18 +5,18 @@ services: docker
 dist: xenial
 
 env:
-  - VERSION=9.4-2.5
-  - VERSION=9.4-2.5 VARIANT=alpine
-  - VERSION=9.5-2.5
-  - VERSION=9.5-2.5 VARIANT=alpine
-  - VERSION=9.6-2.5
-  - VERSION=9.6-2.5 VARIANT=alpine
-  - VERSION=10-2.5
-  - VERSION=10-2.5 VARIANT=alpine
-  - VERSION=11-2.5
-  - VERSION=11-2.5 VARIANT=alpine
   - VERSION=12-2.5
   - VERSION=12-2.5 VARIANT=alpine
+  - VERSION=11-2.5
+  - VERSION=11-2.5 VARIANT=alpine
+  - VERSION=10-2.5
+  - VERSION=10-2.5 VARIANT=alpine
+  - VERSION=9.6-2.5
+  - VERSION=9.6-2.5 VARIANT=alpine
+  - VERSION=9.5-2.5
+  - VERSION=9.5-2.5 VARIANT=alpine
+  - VERSION=9.4-2.5
```

Comment:
9.4 goes EOL on Thursday; consider dropping?

Reply:
good point, and now's a good time to trim down things to be maintained.  Thanks @dbaston